common/models/update_environment_request.go

```diff
@@ -6,16 +6,17 @@ import (
 	swagger "github.com/zpatrick/go-plugin-swagger"
 )
 
+type UpdateEnvironmentRequestJob struct {
+	EnvironmentID string `json:"environment_id"`
```

Comment:
Just commenting on this decision as it runs slightly contrary to what I had previously understood about the implementation. Looks good!

Reply:
😈